.circleci/config.yml

```diff
@@ -44,79 +52,72 @@ commands:
             mkdir ~/.composer
             bash -c 'echo "{\"github-oauth\": {\"github.com\": \"$GITHUB_TOKEN\"}}"' > ~/.composer/auth.json
       - install-ddev
+      - checkout
+      - when:
+          condition: << parameters.upgrade >>
+          steps:
+            - run:
+                name: Switch to dev branch for upgrade testing
+                command: |
+                  git checkout 2.x
+                  # The following should be removed once the 2.x branch has the ddev add-on config.
+                  git checkout $CIRCLE_BRANCH -- .ddev composer.json
```

Comment:
Need to add composer.json because if we build from 2.x w/o recommended project, we don't get admin toolbar. We should remove that as a dependency and fix any tests that expect it.